lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -424,6 +455,9 @@ class FunctionEntrypointAnalyzer {
                                                 const FunctionSummary &,
                                                 const efa::BasicBlock &);
   FunctionSummary importPrototype(model::FunctionType::Values, model::TypePath);
+  void initializeMapForDeductions(FunctionSummary &, abi::RegisterState::Map &);
+  std::optional<abi::RegisterState::Values>
+  tryGetRegState(model::Register::Values, const ABIAnalyses::RegisterStateMap &);
```

Comment:
Do not abbreviate

Reply:
Needed to abbreviate here otherwise check-convention would mess around later:
```cpp
  for (auto &GPRRetVal : GPRRetVals)
    if (auto MaybeState = tryGetRegisterState(GPRRetVal,
                                              Results
                                                .FinalReturnValuesRegisters))
```